general/lecture1/exceptions/NumberOutOfRangeException.java

```diff
@@ -0,0 +1,8 @@
+package general.lecture1.exceptions;
```

Comment:
Your files should be located in the folder `java`, not `general`

Also, please follow the common structure for java projects: 
```
project_root}/java/${your_name}/src/lecture1...
```